templates/frontend/pages/catalog.tpl

```diff
@@ -27,6 +33,20 @@
 	{* Monograph List *}
 	{else}
 		{include file="frontend/components/monographList.tpl" monographs=$publishedMonographs featured=$featuredMonographIds}
+
+		{if $prevUrl || $nextUrl}
+			<div class="cmp_pagination" aria-label="{translate|escape key="catalog.pagination.label"}">
+				{if $prevUrl}
+					<a class="prev" href="{$prevUrl|escape}">{translate key="catalog.pagination.previous"}</a>
```

Comment:
Do these URLs present with proper encoding when `path_info_disabled` is on? OTOH you might be double-escaping.

Reply:
I always forget about the smarty url function. I've implemented it using that now. 77f3c4d